scripts/ci/run_tests.sh

```diff
@@ -7,4 +7,4 @@ fi
 
 echo "Using ${HADOOP_DISTRO} distribution of Hadoop from ${HADOOP_HOME}"
 
-python test/runtests.py -v
+python test/runtests.py -v $@
```

Comment:
Nah, I tried this and it doesn't seem to work. Also, I expected it to not work either. How would the arguments be passed if you just execute `python test/runtests.py -v`?